schemas/specimen.json

```diff
@@ -0,0 +1,466 @@
+          "Sinus(es), maxillary",
+          "Skeletal muscle",
+          "Skin",
+          "Skull",
+          "Small bowel",
+          "Small bowel - mucosa only",
+          "Small finger",
+          "Soft tissue",
+          "Spinal column",
+          "Spinal cord",
+          "Spleen",
+          "Splenic flexure",
+          "Sternum",
+          "Stomach",
+          "Stomach - mucosa only",
+          "Subcutaneous tissue",
+          "Synovium",
+          "Temporal cortex",
+          "Tendon",
+          "Testis",
+          "Thigh",
+          "Thoracic spine",
+          "Thorax",
+          "Throat",
+          "Thumb",
+          "Thymus",
+          "Thyroid",
+          "Tibia",
+          "Tongue",
+          "Tonsil",
+          "Tonsil (pharyngeal)",
+          "Trachea / major bronchi",
+          "Transverse colon",
+          "Trunk",
+          "Umbilical cord",
+          "Ureter",
+          "Urethra",
+          "Urinary tract",
+          "Uterus",
+          "Uvula",
+          "Vagina",
+          "Vas deferens",
+          "Vein",
+          "Venous",
+          "Vertebra",
+          "Vulva",
+          "White blood cells",
+          "Wrist"
+        ]
+      }
+    },
+    {
+      "name": "central_pathology_confirmed",
+      "description": "Indicate whether the histologic description of tissue or cells was confirmed by a pathology review of frozen or formalin fixed slide(s) completed after the diagnostic pathology review of the tumour sample used to extract analyte(s).",
+      "valueType": "string",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Yes",
+          "No",
+          "Unknown",
+          "Not reported"
+        ]
+      }
+    },
+    {
+      "name": "tumour_histological_type",
+      "description": "The code to represent the histology (morphology) of neoplasms that is usually obtained from a pathology report, according to the International Classification of Diseases for Oncology, 3rd Edition (WHO ICD-O-3).",
+      "valueType": "string",
+      "restrictions": {
+        "required": true,
+        "regex": "M-[0-9]{4}/[0-9]{2}$"
+      }
+    },
+    {
+      "name": "tumour_grading_system",
+      "description": "Specify the tumour staging system used to assess the description of a tumour based on how abnormal the tumour cells and the tumour tissue look under a microscope. Tumour grade is an indicator of how quickly a tumour is likely to  grow.",
+      "valueType": "string",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Default",
+          "Gleason",
+          "Nottingham",
+          "Brain cancer",
+          "ISUP",
+          "Lymphoid neoplasms"
+        ]
+      }
+    },
+    {
+      "name": "tumour_grade",
+      "description": "Grade of the tumour as assigned by the reporting tumour_grading_system.",
+      "valueType": "string",
+      "restrictions": {
+        "required": true      }
```

Comment:
This field is currently free-text. Will there be script-based validation to make sure only allowed values are submitted based on what `tumour_grading_system` is used?

Reply:
The `tumour_grade` field is not supposed to be free-text. It should use a controlled vocabulary list depending on the `tumour_grading_system` selected. For example, if "Gleason"  is submitted in the `tumour_grading_system` field, then the `tumour_grade` field will consist of a CV list consisting of Gleason scores.

Reply:
Yes, this should be a script validation eventually.